File: python/guess.py
import random

guess=random.randint(1,100)
#print(guess)
count=10
num=int(input("enter ur guess: "))
    
while (True and count>0):
    if (num==guess):
        print("YOU WON")
        print("you guessed in {} attempts".format(10-count))
        break
    elif (num<guess):
        print("bigger number please")
        count-=1
        #num=int(input("enter ur guess: "))
    elif (num>guess):
        print("lower number please")
        count-=1
    print("u have {} chances left".format(count))
    num=int(input("enter ur guess: "))
   
if (count==0):
    print("YOU LOSE")

File: C language/guess.c
#include<stdio.h>
#include<stdlib.h>
#include<time.h>
int main(){
    int num,guess,nguesses=1;
    srand(time(NULL));
    num = rand()%100 ;  // number b/w 0 to 100
   // printf("%d\n",num);
    do{
        printf("Guess the number between 0 to 100\n");
        scanf("%d",&guess);
        if(guess>num){
            printf("lower number please\n");
        }
        else if(guess<num){
            printf("higher number please\n");
        }
        else{
            printf("you guessed it in %d attempts\n",nguesses);
        }
        nguesses++;
    }while (guess!=num);
     return 0;
}